### ISSUE TITLE
`renorm` arg (and friends) in layer_batch_normalization()

### ISSUE THREAD

**Issue by t-kalinowski** [rstudio/keras#908]
Hi,

`tf.keras.layers.BatchNormalization` gained a `renorm` argument gained a couple TF versions ago, it would be great if that was exposed on the R side from `keras::layer_batch_normalization()`. 

I was also curious if there were other locations where the API's diverged, so I did a "quick & dirty" script to compare the function formals. It suggests there might be some other minor api divergences. In case it's of use, here is code I used to make the comparison (e.g., if you want to do a similar comparison for future major TF releases).

``` r
library(tibble)
library(purrr)
library(dplyr, warn.conflicts = FALSE)


find_first_arg_of_call <- function(expr, call = quote(create_layer)) {
  find_it <- function(expr) {
    if(!is.call(expr)) return(NULL)
    for(e in as.list(expr))
      if(is.call(e) && identical(e[[1]], call))
        return(e[[2]])
      else {
        if(!is.null(it <- find_it(e)))
          return(it)
      }
  }
  find_it(expr)
}


default_ignore <- c("self", "kwargs")

ls(pattern = "^layer_", envir = asNamespace("keras")) %>%
  lapply(function(r_func_nm) {
    r_func <- get(r_func_nm, envir = asNamespace("keras"))
    r_args <- names(formals(r_func))
    py_layer <-
      eval(find_first_arg_of_call(body(r_func)), asNamespace("keras"))
    py_init_args <- py_layer$`__init__`$`__code__`$co_varnames
    py_init_defaults <- py_layer$`__init__`$im_func$func_defaults
    lst(r_func_nm, r_func, r_args, py_layer, py_init_args, py_init_defaults)
  }) %>%
  transpose() %>%
  as_tibble() %>%
  mutate_at("r_func_nm", unlist) %>%
  mutate(
    missing_in_r_func_args = map2_chr(r_args, py_init_args,
                        ~setdiff(.y, c(.x, default_ignore)) %>% paste(collapse = ", "))
  ) %>%
  filter(missing_in_r_func_args != "") %>%
  select(r_func_nm, missing_in_r_func_args) %>%
  print(n = Inf)
#> # A tibble: 19 x 2
#>    r_func_nm            missing_in_r_func_args                             
#>    <chr>                <chr>                                              
#>  1 layer_activation_se… activation                                         
#>  2 layer_batch_normali… renorm, renorm_clipping, renorm_momentum, fused, v…
#>  3 layer_conv_2d_trans… stride, out_pad                                    
#>  4 layer_conv_3d_trans… stride, out_pad                                    
#>  5 layer_conv_lstm_2d   cell                                               
#>  6 layer_cropping_2d    height_cropping, width_cropping                    
#>  7 layer_cropping_3d    dim1_cropping, dim2_cropping, dim3_cropping        
#>  8 layer_cudnn_gru      go_backwards, cell_spec                            
#>  9 layer_cudnn_lstm     go_backwards, cell_spec                            
#> 10 layer_embedding      dtype                                              
#> 11 layer_gru            implementation, cell                               
#> 12 layer_lambda         function, function_args                            
#> 13 layer_locally_conne… implementation                                     
#> 14 layer_locally_conne… implementation                                     
#> 15 layer_lstm           implementation, cell                               
#> 16 layer_max_pooling_1d data_format                                        
#> 17 layer_simple_rnn     cell                                               
#> 18 layer_zero_padding_… height_padding, width_padding                      
#> 19 layer_zero_padding_… dim1_padding, dim2_padding, dim3_padding
```

<sup>Created on 2019-11-01 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by dfalbel** [comment]
Thanks @t-kalinowski
